src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -316,13 +344,18 @@ private void initShuffleboard() {
 					.addDouble("Top FlyWheel Temp", () -> launcherTopMotor.getMotorTemperature());
 		}
 
-		launcherIsAtSpeed =
-				Shuffleboard.getTab("Match")
-						.add("flywheels at target speed", false)
-						.withSize(1, 1)
-						.withWidget(BuiltInWidgets.kBooleanBox)
-						.withPosition(0, 2)
-						.getEntry();
+		Shuffleboard.getTab("Launcher")
+				.addDouble(
+						"relative encoder", () -> launcherAngleOneMotor.getEncoder().getPosition() * 360);
```

Comment:
FYI, looking at logs, this isn't updated when the robot is disabled. I suspect the `.addDouble(..)` way of supplying data only samples the values when the robot is enabled. Nothing needing to change here, just something I noticed from the logs